holohub

```diff
@@ -15,5 +15,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
-PYTHONPATH=${PYTHONPATH}:${SCRIPT_DIR} python3 ${SCRIPT_DIR}/utilities/cli/holohub.py "$@"
+import os
+import sys
+
+sys.path.append(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
could also just append `./utilities/cli` here and import just `holohub` below? just seems risky that this could find any python modules inside holohub nost just this one, but that is consistent with what was done with bash